src/sentry/api/endpoints/project_stacktrace_link.py

```diff
@@ -182,6 +184,40 @@ def sort_code_mapping_configs(
                     sorted_configs.insert(0, config)
         return sorted_configs
 
+    def get_commit_sha(
+        self,
+        integration: Integration,
+        organization_id: str,
+        line_no: int,
+        filepath: str,
+        repository: Repository,
+        branch: str,
+    ) -> Optional[str]:
+        commitSha = ""
+        integrationInstallation = integration.get_installation(organization_id=organization_id)
```

Comment:
Do this call outside of this function and pass `integration_installation` to this function.
Once we move this function into the integration we will not need this.